src/Resources/config/parameters.yml

```diff
@@ -6,3 +6,7 @@ parameters:
     uecode_qpush.provider.sync:          Uecode\Bundle\QPushBundle\Provider\SyncProvider
     uecode_qpush.provider.custom:        Uecode\Bundle\QPushBundle\Provider\CustomProvider
     uecode_qpush.provider.file:          Uecode\Bundle\QPushBundle\Provider\FileProvider
+    uecode_qpush.build_command.class:    Uecode\Bundle\QPushBundle\Command\QueueBuildCommand
+    uecode_qpush.destroy_command.class:  Uecode\Bundle\QPushBundle\Command\QueueDestroyCommand
+    uecode_qpush.publish_command.class:  Uecode\Bundle\QPushBundle\Command\QueuePublishCommand
+    uecode_qpush.receive_command.class:  Uecode\Bundle\QPushBundle\Command\QueueReceiveCommand
```

Comment:
Since Symfony 3.0 the best practices recommend to remove all `*.class` parameters. See the explanations in the issue symfony/symfony#11881. So, it may be wiser not to add new `*.class` parameters, but to retain old parameters for backward compatibility.

Reply:
I missed that. I fixed it.